bin/setup

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+bundle install
+
+read -r -p "Would you like to update your taxonomy data file? [y/N] " response
+
+if [[ $response =~ ^([yY])$ ]]; then
+    # echo "Generating taxonomy data for you, please wait..."
```

Comment:
Assuming this comment isn't intentional. Also the message is a little wordy, `Generating taxonomy data...` is probably sufficient.